tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+# the inclusion of the tests module is not meant to offer best practices for
+# testing in general, but rather to support the `find_packages` example in
+# setup.py that excludes installing the "tests" package
```

Comment:
If ```find_packages``` is only relevant for functionality (tests, documentation) that we deem out of scope of the project, then I'd rather remove use of ```find_packages``` (or at least remove the example of excluding directories).

I'm concerned that the scope of the sample project is reducing to the point that we can't actually demonstrate any meaningful packaging capabilities, but I'd rather we didn't include things that aren't in scope, as otherwise we'll keep getting people who assume we *are* providing recommendations (e.g. the recent thread on test recommendations).